engine/src/main/java/org/terasology/engine/core/subsystem/lwjgl/LwjglDisplayDevice.java

```diff
@@ -88,13 +89,15 @@ public void setDisplayModeSetting(DisplayModeSetting displayModeSetting, boolean
                 break;
             case WINDOWED_FULLSCREEN:
                 GLFWVidMode vidMode = desktopResolution.get();
-                GLFW.glfwSetWindowMonitor(window,
-                        MemoryUtil.NULL,
-                        0,
-                        0,
-                        vidMode.width(),
-                        vidMode.height(),
-                        GLFW.GLFW_DONT_CARE);
+                for(int i=0;i<2;i++){
+                    GLFW.glfwSetWindowMonitor(window,
+                            MemoryUtil.NULL,
+                            0,
+                            0,
+                            vidMode.width(),
+                            vidMode.height(),
+                            GLFW.GLFW_DONT_CARE);
+                }
```

Comment:
```suggestion
                // Attempt to go into fullscreen twice to fix the taskbar showing on-top of the game on Windows.
                // See also: https://github.com/MovingBlocks/Terasology/issues/5228.
                for (int i = 0; i < 2; i++) {
                    GLFW.glfwSetWindowMonitor(window,
                            MemoryUtil.NULL,
                            0,
                            0,
                            vidMode.width(),
                            vidMode.height(),
                            GLFW.GLFW_DONT_CARE);
                }
```

Given that this is a very unusual solution, I think it deserves a comment to explain why it is done this way. The suggestions above should also fix the checkstyle issues.